src/SixLabors.Fonts/FontReader.cs

```diff
@@ -85,13 +85,14 @@ internal FontReader(Stream stream, TableLoader loader)
             this.CompressedTableData = true;
             this.Headers = Woff2Utils.ReadWoff2Headers(reader, tableCount);
 
+            this.disposable = true;
+
             byte[] compressedBuffer = reader.ReadBytes((int)totalCompressedSize);
             var decompressedStream = new MemoryStream();
             using var input = new MemoryStream(compressedBuffer);
             using var decompressor = new BrotliStream(input, CompressionMode.Decompress);
             decompressor.CopyTo(decompressedStream);
             decompressedStream.Position = 0;
-            this.stream.Dispose();
```

Comment:
Yeah, that's a mistake. The parent should never dispose of a field it doesn't own.